### PR TITLE
added worldpopulation script

### PR DESCRIPTION
**Pull request by jacobfrench**
Run worldpopulation.py without any parameters to get a full list of country populations or give it a country code to append only that country to worldpopulation.tsv.

### CODE REVIEW

File: population/World Population/worldpopulation.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+	codes = []
+	with open(codeDir, 'rb') as f:
+		reader = csv.reader(f, delimiter=' ')
+		for row in reader:
+			codes.append(row[0])
+	return codes
+
+
+
+# ---------------------------------------------------------------------------------
```
Comment (sirwart) [inline population/World Population/worldpopulation.py]: The canonical way to structure code like this in Python is to wrap the command-line portion is a
```if __name__ == "__main__":```
This lets you run this file directly as a script and also import the code from another file

File: population/World Population/worldpopulation.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+DATADIR = ROOTDIR + '/data/'
+
+if(len(sys.argv) < 2):
+	countryCodes = getCountryCodes(ROOTDIR + '/read_only/countrycodes.csv')
+else:
+	countryCodes = [str(sys.argv[1])]
+
+
+for code in countryCodes:
+	url = 'http://api.worldbank.org/countries/' + code.upper() +\
```
Comment (sirwart) [inline population/World Population/worldpopulation.py]: It looks like this is giving back data that's not technically countries, such as regions (see "Central Europe and the Baltics") and also the total world population. I think we should just strip out the regions for now and put the world population data in a different file

File: population/World Population/worldpopulation.py (hunk 1 above)
Comment (jacobfrench) [inline population/World Population/worldpopulation.py]: Ok, I'll change it and do them this way from here on out.

File: population/World Population/worldpopulation.py (hunk 2 above)
Comment (jacobfrench) [inline population/World Population/worldpopulation.py]: I'll have to go through the file and get the names of all the regions. There wasn't anything I could use to differentiate between the two in the json files, but I'll take care of it.

**Pull request merged**